### ISSUE TITLE
WARNING:Can't import the 'docker' package. Container scanning functionality will be disabled.

### ISSUE THREAD

**Issue by iamtanmay** [OpenSCAP/openscap-daemon#134]
Hi.

I am on RHEL 7. I yum installed docker

Docker is active and running. systemctl enable docker, systemctl start docker

I yum installed docker-python-1.4.0-115.el7 and I yum installed openscap-utils, to get openscap-docker functionality.

Yet, I keep getting the warning on verbose mode (atomic scan --verbose ....)

"WARNING:Can't import the 'docker' package. Container scanning functionality will be disabled."

What does it mean ? I am pulling my hair out, because I have no idea what to do about this warning.

Thank you for your help.
Tanmay

**Comment by jan-cerny** [comment]
Hi @iamtanmay If you scan container using `atomic` command, this warning should not not affect you, it should be affecting only the low-level `oscap-docker` command.

Does `atomic scan` produce any results?

**Comment by iamtanmay** [comment]
Yes, the results seem to be as expected. Please see below and tell me if it looks wrong:

atomic scan --verbose registry.access.redhat.com/rhscl/nodejs-6-rhel7

gives:

The following issues were found:

     RHSA-2018:0502: kernel-alt security and bug fix update (Important)
     Severity: Important
       RHSA URL: https://access.redhat.com/errata/RHSA-2018:0502
       RHSA ID: RHSA-2018:0502-01
       Associated CVEs:
           CVE ID: CVE-2017-16994
           CVE URL: https://access.redhat.com/security/cve/CVE-2017-16994
           CVE ID: CVE-2017-17712
           CVE URL: https://access.redhat.com/security/cve/CVE-2017-17712

     RHSA-2018:0180: kernel-alt security and bug fix update (Important)
     Severity: Important
       RHSA URL: https://access.redhat.com/errata/RHSA-2018:0180
       RHSA ID: RHSA-2018:0180-02
       Associated CVEs:
           CVE ID: CVE-2017-1000405
           CVE URL: https://access.redhat.com/security/cve/CVE-2017-1000405

     RHSA-2017:0372: kernel-aarch64 security and bug fix update (Important)
     Severity: Important
       RHSA URL: https://access.redhat.com/errata/RHSA-2017:0372
       RHSA ID: RHSA-2017:0372-01
       Associated CVEs:
           CVE ID: CVE-2016-5195
           CVE URL: https://access.redhat.com/security/cve/CVE-2016-5195
           CVE ID: CVE-2016-7039
           CVE URL: https://access.redhat.com/security/cve/CVE-2016-7039
           CVE ID: CVE-2016-8666
           CVE URL: https://access.redhat.com/security/cve/CVE-2016-8666


Many thanks for such a quick reply. I was worried that I had missed an SDK or something.
